salt/utils/vmware.py

```diff
@@ -187,7 +187,6 @@ def get_service_instance(host, username, password, protocol=None, port=None):
     service_instance = GetSi()
     if service_instance:
         if service_instance._GetStub().host == ':'.join([host, str(port)]):
-            service_instance._GetStub().GetConnection()
```

Comment:
@gtmanfred made this change in https://github.com/saltstack/salt/commit/070eaf07f09dd0efbd08f8cd93db4dbb628b2a69 so he needs to review this specific line change. I wasn't aware of any changes made to the connection library.